webview/src/experiments/components/table/styles.module.scss

```diff
@@ -110,6 +110,9 @@ $badge-size: 0.85rem;
 }
 
 .experiments {
+  position: fixed;
+  top: 0;
+  left: 0; // position, top and left are set to fix indicators in storybook
```

Comment:
@sroy3 @julieg18 any issue with adding this to "fix" the stories? There is no change in the production version of the app. Usage is here: https://github.com/iterative/vscode-dvc/blob/main/webview/src/experiments/components/Experiments.tsx#L207-L208

Reply:
The answer is no it breaks scrolling